pkg/api/helper/helpers.go

```diff
@@ -112,6 +112,18 @@ func IsStandardContainerResourceName(str string) bool {
 	return standardContainerResources.Has(str)
 }
 
+var nonstandardContainerResources = sets.NewString(
```

Comment:
The existing `standardContainerResources` includes CPU and memory only. So I just consider other resources to be non-standard. This may be not so accurate.